apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/AssertUtils.java

```diff
@@ -0,0 +1,52 @@
+package gov.cms.bfd.server.war.utils;
+
+import static org.junit.Assert.fail;
+
+import java.util.Objects;
+
+public class AssertUtils {
+
+  @FunctionalInterface
+  public interface Executor {
+    void execute();
+  }
+
+  public static Exception catchExceptions(Executor executor) {
+    Exception ex = null;
+
+    try {
+      executor.execute();
+    } catch (Exception e) {
+      ex = e;
+    }
```

Comment:
When I test for an expected exception I include `fail("expected an exception to the thrown");` to fail the test if none is thrown.

Reply:
It's not really doing an assertion test for an exception per se, it's just catching any exception that might get thrown, the assertion is actually done in the test itself which compares the caught exception (or null if no exception was caught) to the expected one.